app/src/main/java/fr/free/nrw/commons/media/ZoomableActivity.kt

```diff
@@ -133,6 +133,8 @@ class ZoomableActivity : BaseActivity() {
     @Inject
     lateinit var customSelectorViewModelFactory: CustomSelectorViewModelFactory
 
+    private val originLabel = "Origin";
```

Comment:
This value is know at compile time, so better define it as const val I believe, please follow https://www.baeldung.com/kotlin/const-var-and-val-keywords#const

Reply:
Also, please add a Javadoc comment explaining what it is used for.

Reply:
Done